CENNZnet-infrastructures/CENNZnet-browser-extension.md

```diff
@@ -9,10 +11,43 @@ The repo can be found [here](https://github.com/cennznet/extension)
 
 Please install from the [Chrome Web Store](https://chrome.google.com/webstore/detail/cennznet-extension/feckpephlmdcjnpoclagmaogngeffafk).
 
-## User Guide
+## UI User Guide
 The user guide for the CENNZnet browser extension can be found [here](https://medium.com/centrality/cennznet-browser-extension-user-guide-8f4d924a9f12).
 
-## Using the extension with your DApp
+## Using the extension with your DApp through the API hook
+
+Import the extension's API hook:
+```js
+import { UseCennznet, UseCennznetRx } from '@cennznet/api/hooks/UseCennznet';
+```
+
+Use the extension's API hook:
+```js
+const { api, accounts, isExtensionInstalled } = await UseCennznet('app_name', { network: 'azalea' });
+// accounts is the list of accounts available from the extension
+```
+
+The extension's named as ``'cennznet-extension'``.
+
+## Signing transactions through the extension
+```js
+const injector = from(web3FromSource('cennznet-extension');
+const signer = injector.signer;
+
+// Create a transaction
+const transaction = api.tx.balances.transfer('5C7bLoDLAUjasvf2SPrSAZuiybFtgQ5ya5iqMgBeLzkwGRRr', 1000);
```

Comment:
may be lets use module which exist in our chain..
genericAsset over balance..

Reply:
oh true haha thanks :D